front-end/src/screens/Settings/address.tsx

```diff
@@ -183,74 +175,82 @@ const Address = ({ className }: Props): JSX.Element => {
 		);
 	}
 
-	if (accounts.length === 0) {
+	const addressList = (accounts: InjectedAccountWithMeta[], title: string, showOnlyUnlink: boolean, showAccounts: boolean) => {
 		return (
-			<Form className={className} standalone={false}>
+			<>
 				<Form.Group>
 					<Form.Field width={16}>
-						<label>Linked address</label>
+						<label className='header'>{title}</label>
+						<div className='ui list'>
+							{accounts.map(account => {
+								const address = getEncodedAddress(account.address);
+
+								return address &&
+									<Grid key={address}>
+										<Grid.Column width={7}>
+											<div className='item'>
+												<AddressComponent className='item' address={address} extensionName={account.meta.name} />
+											</div>
+										</Grid.Column>
+										<Grid.Column width={3}>
+											<div className='button-container'>
+												<Button
+													className={'social'}
+													negative={showOnlyUnlink ? true : currentUser.addresses?.includes(address) ? true : false}
+													onClick={() => showOnlyUnlink ? handleUnlink(address) : currentUser.addresses?.includes(address) ? handleUnlink(address) : handleLink(address, account)}
+												>
+													{showOnlyUnlink ? unlinkIcon : currentUser.addresses?.includes(address) ? unlinkIcon : linkIcon}
+												</Button>
+											</div>
+										</Grid.Column>
+										<Grid.Column width={6} >
+											{currentUser.addresses?.includes(address) && currentUser.defaultAddress !== address ?
```

Comment:
Same here, please have the `?` and `:` at the beginning of the line.

Reply:
also this is actually again a case where the `null` is not needed, please use `condition && <Component/>`